src/test/kotlin/no/skatteetaten/aurora/gobo/resolvers/application/ApplicationQueryResolverTest.kt

```diff
@@ -54,4 +52,17 @@ class ApplicationQueryResolverTest {
             .jsonPath("$firstApplicationDeployment.namespace.name").isNotEmpty
             .jsonPath("$firstApplicationDeployment.details.buildTime").isNotEmpty
     }
+
+    @Test
```

Comment:
Tester vi rammeverket her, tro? Eller verifiserer du at vi ikke har "ødelagt" noe med custom config?

Reply:
Jeg tror jeg fjerner den, den var i utgangspunktet laget for exploratory testing. Hva skjer hvis jeg sender inn en tom query.